Jenkinsfile

```diff
@@ -72,19 +74,6 @@ pipeline {
                 }
             }
         }
-        stage('Check QA Site') {
-            when { changeRequest(); }
-            steps {
-                script {
```

Comment:
Do we actually need this?

Reply:
I think we can drop the  QA_SITE_* stuff that doesn't do anything, but I would lean towards leaving the curl command in (even if we just moved it to the last line of the prior stage), since it's a simple enough check and will alert us to issues earlier.

Reply:
Going to restore simplified version